src/types/base.rs

```diff
@@ -55,6 +56,22 @@ impl<'a> PactType<'a> {
                     buf.push(b.swap_bits())
                 }
             }
+            PactType::List(l) => {
+                let mut buf_elements: Vec<u8> = Vec::<u8>::default();
+                for element in l {
+                    match element {
+                        PactType::StringLike(_) => element.encode(&mut buf_elements),
+                        PactType::Numeric(_) => element.encode(&mut buf_elements),
```

Comment:
Question: What's the difference between the 2 cases? i.e. it looks like we are always calling the "encode" function

Reply:
There isn't - we want to encode both of them 😄 
I don't think match allows fallthrough like a switch-case statement.